modules/microshift-cni-multus-bridge-example-config.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-bridge-example-config_{context}"]
+= Example configuration for the bridge additional network
+
+Use the following procedure to configure a bridge-type additional network. You can use the example workflow here that uses `host-local` IP address management (IPAM) to configure other supported additional network types.
+
+[IMPORTANT]
+====
+If you use `bridge` and the `dhcp` IPAM, a DHCP server listening on the bridged network is required. If you are also using a firewall, configuring the firewalld service to allow DHCP traffic on the network zone is also required. You can run the `firewall-cmd --remove-service=dhcp` command in this case.
+====
+
+.Prerequisites
+
+* The Multus CNI is installed.
+* The OpenShift CLI (`oc`) is installed.
+* The cluster is running.
+
+.Procedure
+
+. Optional: Verify that the {microshift-short} cluster is running and includes the Multus CNI by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-multus
+----
++
+.Example output
+[source,terminal]
+----
+NAME                READY   STATUS    RESTARTS   AGE
+dhcp-daemon-dfbzw   1/1     Running   0          5h
+multus-rz8xc        1/1     Running   0          5h
+----
+
+. Create a `NetworkAttachmentDefinition` configuration file by running the following command and using the following example file for reference:
++
+[source,terminal]
+----
+$ oc apply -f ./test/assets/multus/bridge-nad.yaml
+----
++
+.Example `NetworkAttachmentDefinition` file
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-conf
+spec:
+  config: '{
+      "cniVersion": "0.4.0",
+      "type": "bridge", <1>
+      "bridge": "br-test", <2>
+      "mode": "bridge",
+      "ipam": {
+        "type": "host-local", <3>
+        "ranges": [
+          [
+            {
+              "subnet": "10.10.0.0/24",
+              "rangeStart": "10.10.0.20",
+              "rangeEnd": "10.10.0.50",
+              "gateway": "10.10.0.254"
+            }
+          ]
+        ],
+        "dataDir": "/var/lib/cni/br-test"
+      }
+    }'
+----
+<1> The `type` value specifies a name of the CNI plugin. In this example, "bridge" is used.
+<2> The `bridge` value is name of the bridge on the {microshift-short} host that is used. The additional interface of the pod is connected to that bridge. In this example, the name is "br-test".
```

Comment:
Do we want to mention that if the interface doesn't exist on the host, the bridge CNI will create it, but if it exists, it will be reused?